cv.md

```diff
@@ -0,0 +1,51 @@
+# Gevindu Dinith Hapuarachchi
+### Intern Software Engineer
```

Comment:
One rule about headings to follow later as well - it is a good practise to add them sequentially, so: 
"#"
"##"
"###"
"##"
"###"

Instead of:
"#"
"###"
"##"

Reply:
@vieraboschkova Changed